packages/calling-stateful-client/src/CallClientState.ts

```diff
@@ -489,6 +489,11 @@ export interface CallState {
    * Proxy of {@link @azure/communication-calling#CapabilitiesFeature}.
    */
   capabilitiesFeature?: CapabilitiesFeatureState;
+  /* @conditional-compile-remove(hide-attendee-name) */
+  /**
+   * Hide attendee names in teams meeting
+   */
+  hideAttendeeNames?: boolean;
```

Comment:
this will likely update when the calling sdk property rolls in so this is a future suggestion and can ignoree for this pr, but maskAttendeeNames is slightly better as we can't hide them completely

Reply:
Yeah once sdk changes are added, will update the name